boards/genericCH592F.json

```diff
@@ -21,7 +21,8 @@
     "svd_path": "CH59Xxx.svd"
   },
   "frameworks": [
-    "noneos-sdk"
+    "noneos-sdk",
+    "freertos"
```

Comment:
Why only CH592F? Other CH592x are unsupported?

Also, changes to board files should go through the Python script that generates them, otherwise they will be lost with the next update.

E.g. after

https://github.com/Community-PIO-CH32V/platform-ch32v/blob/d9663011522ffa485b465a2dcdcebafa3970bcd1/misc/scripts/gen_boarddefs.py#L290-L292

```py
    if chip_l.startswith("ch59"):
        base_json["frameworks"].append("freertos")
```
then reexecute the script.

Reply:
I tested only ch592F but actually the SDK is the same for CH591 and CH592X so it should work. I made the changes in latest commit